packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -641,11 +641,6 @@ class GoTrueClient {
             'No session found for the auth code.');
       }
 
-      if (storeSession == true) {
-        _saveSession(session);
-        notifyAllSubscribers(AuthChangeEvent.signedIn);
-      }
-
```

Comment:
The same set of code can be found in the `exchangeCodeForSession()` function [here](https://github.com/supabase/supabase-flutter/blob/main/packages/gotrue/lib/src/gotrue_client.dart#L318), so this codeblock is unnecessary.